collections/Export/Export Transactions Data CSV.bru

```diff
@@ -0,0 +1,27 @@
+meta {
+  name: Export Transactions Data CSV
+  type: http
+  seq: 2
+}
+
+get {
+  url: {{mpm_backend_url}}/api/export/transactions/demo_company_admin@example.com?format=csv
+  body: json
+  auth: inherit
+}
+
+params:query {
+  format: csv
+}
+
+body:json {
+  {
+    "deviceType": "meter",
+    "serial_number": "a98c04c2-470c-31c1-8b6a-c32e038b436b",
+    "provider": "wave_money_transaction",
+    "status": 1,
+    "currency": "TSZ",
+    "timeZone": "UTC"
+  }
```

Comment:
it's a `get`, probably should have a `body` right?

Reply:
This look like a collection for a `GET` request to `/api/export/transactions/`... Am I understanding correctly that we are using a payload here (i.e. `body` field) ?

If so:

I know it's technically possible to use `body` in a `GET` request. But by convention, we don't use request body in `GET` requests. It's even discouraged in https://datatracker.ietf.org/doc/html/rfc7231#section-4.3.1

>   A payload within a GET request message has no defined semantics;
   sending a payload body on a GET request might cause some existing
   implementations to reject the request.

Reply:
Thanks for clarifying. We can either make it a post request or use query params. 

What do you suggest?

Reply:
I'd say query params would be most intuitive here. As we indeed "retrieving" information.